app/src/main/java/org/fossasia/phimpme/utilities/RetrofitClient.java

```diff
@@ -0,0 +1,28 @@
+package org.fossasia.phimpme.utilities;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/** Created by @codedsun on 19/Oct/2019 */
+
+// Retrofit client used for API Calls
+public class RetrofitClient {
+
+  private static OkHttpClient.Builder httpClient() {
+    OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
+    HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
+    interceptor.level(HttpLoggingInterceptor.Level.BODY);
+    httpClient.interceptors().add(interceptor);
+    return httpClient;
+  }
+
+  public static Retrofit getRetrofitClient(String baseUrl) {
+    return new Retrofit.Builder()
```

Comment:
Cache it

Reply:
@iamareebjamal for how much time?

Reply:
Forever

Reply:
@iamareebjamal - Cache the response forever?

Reply:
Sorry, I thought it was returning a static instance

Reply:
Will update the PR tomorrow and continue my work for Pintrest PR.